.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 *.log
 **/.DS_Store
 .env
+app.yaml
```

Comment:
We don't want to commit this because we declare the environment variable `ANVIL_API_KEY` in here for the App Engine deploy. We leverage templating instead to create this on the fly right before a deploy.